nifi-extension-bundles/nifi-protobuf-bundle/nifi-protobuf-services/src/main/java/org/apache/nifi/services/protobuf/StandardProtobufReader.java

```diff
@@ -170,6 +174,84 @@ protected PropertyDescriptor buildSchemaTextProperty() {
         return PROTOBUF_SCHEMA_TEXT;
     }
 
+    @Override
+    protected Collection<ValidationResult> customValidate(final ValidationContext validationContext) {
+        final List<ValidationResult> problems = new ArrayList<>(super.customValidate(validationContext));
```

Comment:
Results can also indicate a valid status, so the variable name should be changed.
```suggestion
        final List<ValidationResult> results = new ArrayList<>(super.customValidate(validationContext));
```